x-pack/plugins/triggers_actions_ui/public/application/app.tsx

```diff
@@ -23,11 +24,14 @@ import { TypeRegistry } from './type_registry';
 import { AlertDetailsRouteWithApi as AlertDetailsRoute } from './sections/alert_details/components/alert_details_route';
 import { ChartsPluginStart } from '../../../../../src/plugins/charts/public';
 import { DataPublicPluginStart } from '../../../../../src/plugins/data/public';
+import { PluginStartContract as AlertingStart } from '../../../alerting/public';
 
 export interface AppDeps {
   dataPlugin: DataPublicPluginStart;
   charts: ChartsPluginStart;
   chrome: ChromeStart;
+  alerting: AlertingStart;
```

Comment:
To handle change to optional plugin:
```suggestion
  alerting?: AlertingStart;
```